freqtrade/data/btanalysis.py

```diff
@@ -245,8 +245,10 @@ def delete_backtest_result(file_abs: Path):
     # *.meta.json
     logger.info(f"Deleting backtest result file: {file_abs.name}")
     file_abs_meta = file_abs.with_suffix(".meta.json")
+    file_abs_market_change = file_abs.with_name(file_abs.stem + "_market_change.feather")
```

Comment:
Overall - great enhancement.
practically - if you run backtesting with `--export signals` - it's still leaving artifacts behind - so i've updated this PR to include all files that start with the file_abs.stem